make/hotspot/lib/CompileGtest.gmk

```diff
@@ -1,5 +1,5 @@
 #
-# Copyright (c) 2016, 2018, Oracle and/or its affiliates. All rights reserved.
+# Copyright (c) 2016, 2021/, Oracle and/or its affiliates. All rights reserved.
```

Comment:
I think a / sneaked in here. Please fix. 😄

Reply:
Doh, thanks for catching.